samples/boards/nrf/nrfx_prs/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -1,14 +1,50 @@
+&pinctrl {
+	spi1_default_alt: spi1_default_alt {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 0, 6)>,
+				<NRF_PSEL(SPIM_MOSI, 0, 7)>;
+		};
+		group2 {
+			psels = <NRF_PSEL(SPIM_MISO, 0, 25)>;
+			bias-pull-down;
+		};
+	};
+
+	spi1_sleep_alt: spi1_sleep_alt {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 0, 6)>,
+				<NRF_PSEL(SPIM_MOSI, 0, 7)>,
+				<NRF_PSEL(SPIM_MISO, 0, 25)>;
+			low-power-enable;
+		};
+	};
+
+	spi2_default_alt: spi2_default_alt {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 1, 9)>,
+				<NRF_PSEL(SPIM_MOSI, 1, 8)>,
+				<NRF_PSEL(SPIM_MISO, 1, 7)>;
```

Comment:
MISO needs pull-down here. Previously it was activated in the sample code by:
```
spim_config.miso_pull = NRF_GPIO_PIN_PULLDOWN;
```
Same applies to `nrf9160dk_nrf9160.overlay`.